src/main/java/com/spring/db/entity/address/Address.java

```diff
@@ -0,0 +1,25 @@
+package com.spring.db.entity.address;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
```

Comment:
modify configurations to save `Address` data into table with a name `address` and fields should be: (id, city, address)